pkg/operator/operator/k8s.go

```diff
@@ -294,10 +325,12 @@ func TensorFlowPredictorContainers(api *spec.API) ([]kcore.Container, []kcore.Vo
 	return containers, volumes
 }
 
-func ONNXPredictorContainers(api *spec.API) []kcore.Container {
+func ONNXPredictorContainers(api *spec.API) ([]kcore.Container, []kcore.Volume) {
 	resourceList := kcore.ResourceList{}
 	resourceLimitsList := kcore.ResourceList{}
```

Comment:
nit: rename `resourceList` and `resourceLimitsList` to `apiResourceList` and `apiResourceLimitsList`.